tests/test_queuing.py

```diff
@@ -0,0 +1,368 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the :mod:`pennylane` :class:`QueuingContext` class.
+"""
+
+import pytest
+import pennylane as qml
+import numpy as np
+from pennylane import QueuingContext
+
+
+@pytest.fixture(scope="function")
+def mock_queuing_context(monkeypatch):
+    """A mock instance of the abstract QueuingContext class."""
```

Comment:
Isn't this essentially just the `Queue` class, but made via monkeypatching as opposed to inheritance?

Reply:
This was left over from previous code. I've now removed it, since all tests that used it are now duplicated by testing the explicit `Queue` class